requirements.txt

```diff
@@ -1,8 +1,7 @@
 fastapi==0.100.0
 fastapi-utils==0.2.1
 hjson
-numpy
-numpy-stl
+numpy==1.26.4
```

Comment:
1. The 3D renderer depends on numpy-stl.
2. I believe you can specify a "compatible" version by using `~=`, see https://stackoverflow.com/a/39590286